index.js

```diff
@@ -13,7 +13,8 @@ const errorMsgs = {
     2: 'Invalid Inspect Link Structure',
     3: 'You may only have one pending request at a time',
     4: 'Valve\'s servers didn\'t reply in time',
-    5: 'Valve\'s servers appear to be offline, please try again later'
+    5: 'Valve\'s servers appear to be offline, please try again later',
+    6: 'Error on CSGOFloat API, please try again'
```

Comment:
I think this should be something a bit more quirky like 'Something went wrong on our end, please try again'.